### PR TITLE
feat: add header, footer, style

### PR DESCRIPTION
**Pull request by shanberg**
- Restyled form
- Added Grey Matter header
- Added Grey Matter footer

screen shot 2019-02-25 at 1 32 29 pm

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -17,99 +17,468 @@
...
+
+    function enable() {
+      document.getElementById("username").readonly = false;
+      document.getElementById("password").readonly = false;
+      document.getElementById("newPassword").readonly = false;
+      document.getElementById("newPasswordConfirm").readonly = false;
+      document.getElementById("button").disabled = false;
+    }
+  </script>
 </head>
```
Comment (tilleryd) [inline static/index.html]: Please add a title element with "Grey Matter | LDAP Login"

File: static/index.html [hunk 2]
```diff
@@ -17,99 +17,473 @@
...
+    }
+
+    /* App Footer – cloned from Grey Matter UI Component Library */
+    .app-footer {
+      -webkit-user-select: none;
+      overflow-x: hidden;
+      overflow-y: hidden;
+      padding-top: 0px;
+      padding-right: 4px;
+      font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen,
```
Comment (tilleryd) [inline static/index.html]: font-family should inherit from the html selector defined earlier.

File: static/index.html [hunk 3]
```diff
@@ -17,99 +17,473 @@
...
+      font-weight: 500;
+      margin: auto 0;
+    }
+    .logo {
+      height: 100%;
+      display: inline-block;
+      width: 20px;
+      margin: 0;
+      flex: 0 0 20px;
+      height: 20px;
```
Comment (tilleryd) [inline static/index.html]: redundant height property

File: static/index.html [hunk 4]
```diff
@@ -17,99 +17,473 @@
...
+      background-repeat-x: initial;
+      background-repeat-y: initial;
+      background-attachment: initial;
+      background-origin: initial;
+      background-clip: initial;
+      background-color: rgb(255, 255, 255);
+      font-size: 14px;
+      height: auto;
+      min-height: 32px;
+      padding: 0 16px;
```
Comment (tilleryd) [inline static/index.html]: please combine the padding property into one; there are 3 in this selector

File: static/index.html [hunk 5]
```diff
@@ -17,99 +17,473 @@
...
+    }
+    .links {
+      flex-grow: 1;
+      flex-shrink: 0;
+      flex-basis: auto;
+      text-align: right;
+      padding-top: 0px;
+      padding-right: 0px;
+      padding-bottom: 4px;
+      padding-left: 8px;
```
Comment (tilleryd) [inline static/index.html]: please use `padding: 0 0 4px 8px`

File: static/index.html [hunk 6]
```diff
@@ -365,7 +360,8 @@ <h1>Create a New Password</h1>
     </a>
     <p class="copyright">
       <span
-        >TM and &copy; <span id="footer-year"></span> Decipher Technology Studios</span
+        >TM and &copy; <span id="footer-year"></span> Decipher Technology
+        Studios</span
```
Comment (tilleryd) [inline static/index.html]: This line break is rendering

Comment (tilleryd) [reply static/index.html]: I had to edit the file with `vi` rather than changing my formatter settings.

Comment (shanberg) [reply static/index.html]: Fixed

**Pull request merged**